### PR TITLE
Fix SetChannelCoupling for MODEL_SIGLENT_SDS1000

### PR DESCRIPTION
**Pull request by mandl**
Fix SetChannelCoupling for MODEL_SIGLENT_SDS1000
Fix SetChannelOffset   for MODEL_SIGLENT_SDS1000

@mubes  can you please check. Thanks Stefan

### CODE REVIEW

File: scopehal/SiglentSCPIOscilloscope.cpp [hunk 1]
```diff
@@ -2287,7 +2342,7 @@ void SiglentSCPIOscilloscope::SetSampleDepth(uint64_t depth)
 	{
 		// --------------------------------------------------
 		case MODEL_SIGLENT_SDS1000:
-		    // we can not change memory size in Run/Stop mode
+			// we can not change memory size in Run/Stop mode
```
[the remark this answers is not in the document]
Comment (mubes) [reply scopehal/SiglentSCPIOscilloscope.cpp:2345]: Other than this observation I have no issue with this. It's one of a whole sequence of these micro-fixes we should expect as the SDS1000 support solidifies.

[the remark this answers is not in the document]
Comment (azonenberg) [reply scopehal/SiglentSCPIOscilloscope.cpp:2345]: Will merge as soon as that's fixed.

[the remark this answers is not in the document]
Comment (mandl) [reply scopehal/SiglentSCPIOscilloscope.cpp:2349]: @mubes - ok this is fixed

[the remark this answers is not in the document]
Comment (mubes) [reply scopehal/SiglentSCPIOscilloscope.cpp:2349]: Hi, yep, I'm OK with this. DAVE

**Pull request merged**